segmentstore/server/src/main/java/io/pravega/segmentstore/server/reading/FutureReadResultEntryCollection.java

```diff
@@ -46,19 +46,20 @@
 
     //region AutoCloseable Implementation
 
-    @Override
-    public void close() {
+    List<FutureReadResultEntry> close() {
```

Comment:
I'm wondering if it wouldn't be cleaner to get the list of `FutureReadResultEntry` items separately instead of changing the return value of `close()` and making it not autocloseable.

Reply:
Fixed.